src/app/file/file.cpp

```diff
@@ -936,6 +941,11 @@ void FileOp::operate(IFileOpProgress* progress)
     }
 #endif
 
+    if (m_document && m_document->isReadOnly()) {
+      setError("Cannot save/overwrite a document marked as read-only\n");
```

Comment:
Change to (also this could be in strings/en.ini file)
```suggestion
      setError("Cannot save/overwrite a read-only sprite. Use File > Save As option.\n");
```